qfs/chain.json

```diff
@@ -0,0 +1,77 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "qfs",
+    "status": "live",
+    "network_type": "mainnet",
+    "website": "https://qfs.com/",
+    "pretty_name": "qfs",
+    "chain_id": "qfs-1",
+    "bech32_prefix": "qfs",
+    "daemon_name": "qfs",
+    "node_home": "$HOME/.ssc",
+    "key_algos": [
+      "secp256k1"
+    ],
+    "slip173": "qfs",
```

Comment:
This is not an accepted property (the same data is provided as bech32_prefix)

Reply:
@JeremyParish69 To understand why slip173 "qfs" and bech32_prefix are not the same, I need your assistance.

Reply:
they pretty much are the same, but SLIP173 isn't defined in the chain.json schema. Just use bech32_prefix

Reply:
@JeremyParish69  Thanks for the help . I will remove SLIP173 from the entire chain.json schema.